### DISCUSSION TITLE
Is the test_data.py file is fixed in the backend?

### DISCUSSION THREAD

**Discussion by GuibinChen92057** [PaddlePaddle/PaddleSpatial#251] [BaiduKDDCup2022] [answered]
As the question title, I requires to adjust the TestData class defined in test_data.py, could I re-write the definition of TestData class? thx!

**Comment by shaido987** [comment, marked as answer]
The `TestData` class in test_data.py is fixed and this file can not be adjusted in a normal submission. But you don't have to use that specific file. 

What you could do is copy the file, adjust the code, and import this new class in your code. For example, call the new file test_data_adjusted.py and the new class `TestDataNew` then in your predict file you use something like `from test_data_adjusted import TestDataNew`. Just remember to include the new file in the submission as well.

(note that I'm just a participant and not one of the organizers so this is not an official response.)
